__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+const db = require("../db/connection.js");
+const request = require("supertest");
+const app = require("../db/app.js");
+beforeEach(() => seed(data));
+//console.log(shopData + "this is shop data");
+afterAll(() => db.end());
+
+describe("NC News Topics", () => {
+  test("200 message: returns topics objects in an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        topics.forEach((topic) => {
```

Comment:
can we add a check for the length of the array to make sure we aren't getting any false positives